docs-archive/apache-airflow/2.1.4/_api/airflow/exceptions/index.html

```diff
@@ -567,10 +567,10 @@
             </div>
         </div>
     </div>
-
+        
         <div class="d-flex">
-
-
+            
```

Comment:
Some space have been added? does not matter really but just saying :)

Reply:
No idea where from :(

Reply:
Maybe a new sphinx version ?

Reply:
There are "real" changes too:

https://github.com/apache/airflow-site/pull/481/files#diff-efbd11b756e4cd17f684fcb9773e97cc9fe67456920076a2633d57c216a3976aR815